hack/integration-test/validate.sh

```diff
@@ -16,6 +16,3 @@ if [[ -z "$url" ]]; then
   echo "ERROR: configurationImageURL on ImageClusterInstall ibi-test/ibi-test was not set within 60 seconds"
```

Comment:
We're not really checking this any more. Can we update these messages to reflect that we're checking the data image not the ICI?

Reply:
This doesn't really apply to this change so we can do this in some other PR